tests/baselines/reference/typeAliasDeclareKeywordNewlines.errors.txt

```diff
@@ -0,0 +1,18 @@
+typeAliasDeclareKeywordNewlines.ts(4,1): error TS1142: Line break not permitted here.
+
+
+==== typeAliasDeclareKeywordNewlines.ts (1 errors) ====
+    var declare, type, T;
+    
+    declare type /*unexpected newline*/
+    T1 = null;
+    ~~
+!!! error TS1142: Line break not permitted here.

```

Comment:
I don't like the placement of the diagnostic, but I don't know TypeScript diagnostic conventions. Is here fine or should it point to the `type` keyword on the previous line?

Reply:
You can give a specialized diagnostic, but I think this is actually fine as-is.